js/src/index.ts

```diff
@@ -1,10 +1,10 @@
-import { Composio } from "./sdk";
+import { Composio } from "./sdk/index";
```

Comment:
The import paths in index.ts have been updated to use explicit file extensions, but this could cause issues with module resolution. Consider:

1. Using path aliases in tsconfig.json instead:
```json
{
  "compilerOptions": {
    "baseUrl": ".",
    "paths": {
      "@sdk/*": ["src/sdk/*"],
      "@frameworks/*": ["src/frameworks/*"],
      "@env/*": ["src/env/*"]
    }
  }
}
```

2. Then update imports to use aliases:
```typescript
import { Composio } from "@sdk/index";
import { Workspace } from "@env/index";
```

This makes imports more maintainable and avoids potential resolution issues.